src/System.Net.Http/tests/FunctionalTests/DiagnosticsTests.cs

```diff
@@ -43,7 +43,7 @@ public static void EventSource_ExistsWithCorrectId()
         /// This test must be in the same test collection as any others testing HttpClient/WinHttpHandler
         /// DiagnosticSources, since the global logging mechanism makes them conflict inherently.
         /// </remarks>
-        [OuterLoop] // TODO: Issue #11345
+        [OuterLoop("Uses external server")]
```

Comment:
probably doesn't matter now you have the PR up but generally for backports we would not include unnecessary changes, so the diff is more obvious.